controller-api/src/main/java/com/yahoo/vespa/hosted/controller/api/integration/LogEntry.java

```diff
@@ -30,7 +44,7 @@ public long id() {
         return id;
     }
 
-    public long at() {
+    public Instant at() {
```

Comment:
Are you sure this won't break clients of "controller-api"?

Reply:
Yes.